html-web-page/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HTML Web Page</title>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+      }
+
+      td:not(:first-child) {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <!-- your HTML code here 👇 -->
+    <h1>Jelly of the Month Club</h1>
+    <h2>Make all your friends jelly of your stock of jelly!</h2>
+    <p>
+      Here at JOTMC Inc., we are constantly striving to supplyy our customers
+      witht the highest-quality and most unique jellies on a monthly basis. For
+      the jelly lover in your life (or yourself!),this is the gift that keeps on
+      giving all year round. We offer three tiers of monthly jelly joy, and each
+      one offers their own perks. Here is a breakdown of each, as well as your
+      monthly price:
+    </p>
+    <table>
+      <thead>
+        <tr>
+          <th colspan="2"></th>
+          <th>Basic Plan</th>
+          <th>Deluxe Plan</th>
+          <th>Jelly Giant Plan</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Three new curated jellies per month</td>
+          <td colspan="2">✓</td>
+          <td colspan="1">✓</td>
+          <td colspan="2">✓</td>
+        </tr>
+        <tr>
+          <td>Monthy jellies come with commemorative mason jars</td>
+          <td colspan="2"></td>
+          <td colspan="1">✓</td>
+          <td colspan="2">✓</td>
+        </tr>
+
+        <tr>
+          <td>Free bonus jar of preserves bi-monthly</td>
+          <td colspan="2"></td>
+          <td colspan="1">✓</td>
+          <td colspan="2">✓</td>
+        </tr>
+        <tr>
+          <td>Free yearly tickets to Jelly Con</td>
+          <td colspan="2"></td>
+          <td colspan="1"></td>
+          <td colspan="2">✓</td>
+        </tr>
+
+        <tr>
+          <td>Monthly Pricing</td>
+          <td colspan="2">$30</td>
+          <td colspan="1">$50</td>
+          <td colspan="2">$60</td>
+        </tr>
+      </tbody>
+    </table>
+    <p>Like what you see? Become a part of the JOTMC family today!</p>
+    <form>
+      <label for="name"></label>
+      <input type="text" name="name" id="name" placeholder="Your name" />
+      <label for="plan"></label>
```

Comment:
As we touched on in the Q&A, we don't need to include a `label` if there's no text inside of it.